alembic/versions/b74cc308b1ec_move_responses_to_db.py

```diff
@@ -0,0 +1,83 @@
+revision = 'b74cc308b1ec'
+down_revision = '66d78d1a29ee'
+branch_labels = None
+depends_on = None
+
+import alembic
+import sqlalchemy
+import json
+from collections import defaultdict
+from www.commands import build_response_dict
+
+def upgrade():
+	commands = alembic.op.create_table("commands",
+		sqlalchemy.Column("id", sqlalchemy.Integer, primary_key=True, autoincrement=True),
+		sqlalchemy.Column("access", sqlalchemy.Integer, nullable=False),
+	)
+
+	aliases = alembic.op.create_table("commands_aliases",
+		sqlalchemy.Column("id", sqlalchemy.Integer, primary_key=True, autoincrement=True),
+		sqlalchemy.Column("command_id", sqlalchemy.Integer,
+			sqlalchemy.ForeignKey("commands.id", ondelete="CASCADE"), nullable=False),
+		sqlalchemy.Column("alias", sqlalchemy.String(255), nullable=False),
```

Comment:
[It's recommended to not use length limits on text columns.](https://wiki.postgresql.org/wiki/Don%27t_Do_This#Don.27t_use_varchar.28n.29_by_default)
```suggestion
		sqlalchemy.Column("alias", sqlalchemy.String, nullable=False),
```